inspirehep/dojson/experiments/rules.py

```diff
@@ -48,56 +48,56 @@ def date_started(self, key, value):
             self['date_started'] = val.get('s')
         if val.get('t'):
             self['date_completed'] = val.get('t')
+        if val.get('c'):
+            self['date_cancelled'] = value.get('c')
```

Comment:
Ah well :coffee: 
Also I changed the setup.py locally but I didn't commit it it in the PR